Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -699,6 +699,8 @@ search=Suche
 start=Start
 startDate=Startdatum
 startIndexing=Indexierung starten
+startIndexingAll=Indexierung starten f\u00FCr allen
```

Comment:
Maybe better translation would be `Indexierung \u00FCber alles starten`

Reply:
What about: "Komplette Indexierung starten"